ext/reflection/tests/ReflectionMethod_hasPrototype_basic.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+public ReflectionMethod ReflectionMethod::hasPrototype ( void );
+--CREDITS--
+ollieread - <code@ollie.codes>
```

Comment:
We don't use CREDITS sections AFAIK. They were used when it wasn't possible to keep committer info (because SVN)

Reply:
Ah, no worries. I only have this because I copied the getPrototype one, will remove.

Reply:
Done @Girgias